project/plugins.sbt

```diff
@@ -3,6 +3,7 @@ logLevel := Level.Warn
 addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.6.1")
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.4")
 addSbtPlugin("com.jsuereth" % "sbt-pgp" % "1.0.0")
+addSbtPlugin("com.thoughtworks.sbt-api-mappings" % "sbt-api-mappings" % "1.0.0")
```

Comment:
This was the easiest solution to enable linking to stdlib types from oslo's Scaladoc.